modules/core/src/core/transform.js

```diff
@@ -160,13 +217,23 @@ export default class Transform {
 
         if (this._buffersCreated[feedbackBufferName]) {
           this._buffersCreated[feedbackBufferName].delete();
-          this._buffersCreated[feedbackBufferName] = buffer;
         }
+        this._buffersCreated[feedbackBufferName] = buffer;
         this.feedbackBuffers[current][feedbackBufferName] = buffer;
       }
     }
   }
 
+  // Create a buffer and add to list of buffers to be deleted.
+  _createNewBuffer(name, opts) {
+    const buffer = new Buffer(this.gl, opts);
+    if (this._buffersCreated[name]) {
```

Comment:
`buffersCreated`?  Maybe there is a better name... Can this be called `this._bufferCache`? If not, follow verb-noun convention `this._createdBuffers`?

Reply:
Re-named.